### PR TITLE
SDIT-1546 Handle alerts created due to a new booking

### PR DESCRIPTION
**Pull request by andymarke**
These are identified with the NOMIS standard copy table audit text. When this happens the alert mapping needs updating to point to the new booking. There are no changes required in DPS

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsNomisApiService.kt [hunk 1]
```diff
@@ -15,7 +15,7 @@ import java.time.LocalDate
 class AlertsNomisApiService(@Qualifier("nomisApiWebClient") private val webClient: WebClient) {
   suspend fun getAlert(bookingId: Long, alertSequence: Long): AlertResponse = webClient.get()
     .uri(
-      "/prisoner/booking-id/{bookingId}/alerts/{alertSequence}",
+      "/prisoners/booking-id/{bookingId}/alerts/{alertSequence}",
       bookingId,
```
Comment (andymarke) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsNomisApiService.kt:19]: been meaning to correct these for ages ..

Comment (andymarke) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsNomisApiService.kt:19]: (both version exists right now)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt [hunk 2]
```diff
@@ -40,31 +40,47 @@ class AlertsSynchronisationService(
     if (nomisAlert.audit.auditModuleName == "DPS_SYNCHRONISATION") {
       telemetryClient.trackEvent("alert-synchronisation-created-skipped", telemetry)
     } else {
-      val mapping = mappingApiService.getOrNullByNomisId(event.bookingId, event.alertSeq)
-      if (mapping != null) {
-        telemetryClient.trackEvent(
-          "alert-synchronisation-created-ignored",
-          telemetry + ("dpsAlertId" to mapping.dpsAlertId),
-        )
+      if (nomisAlert.audit.auditAdditionalInfo == "OMKCOPY.COPY_BOOKING_DATA") {
```
Comment (mikehalmamoj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt:43]: Would this be easier to understand with and `else if`? I'm getting a little lost

**Review by mikehalmamoj — APPROVED** [review]

File: src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt (hunk 2 above)
Comment (andymarke) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt:43]: Maybe I will extract into methods

**Pull request merged**